### PR TITLE
Adds reactstrap and removes jquery from webpack

### PR DESCRIPTION
**Pull request by Unesdala**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6846,7 +6861,7 @@ jest@latest:
     import-local "^3.0.2"
     jest-cli "^25.1.0"
 
-jquery@latest:
+jquery@^3.4.1:
```
Comment (JoshuaVSherman) [inline yarn.lock]: @Unesdala please remove it from package.json and reinstall

**Comment by Unesdala** [comment]
@JoshuaVSherman Removed bootstrap and jquery. Webpack won't build with bootstrap installed but jquery not, given bootstrap relies on jquery being installed.

Not 100% sure how well reactstrap will work without it, but there's nothing specifically relying on it right now AFAIK. Even when I redo the layout, the only thing I'd need it for is the mobile menu, and we've got something else for that as is right now, so might not even need it for that.

Either way, gonna do some testing to see how well everything plays with just reactstrap on my site, given it does rely on it for the mobile menu and can go from there.

**Comment by JoshuaVSherman** [comment]
@Unesdala no you need to keep bootrap in the package.json and also import it into the main.scss file, like
@import "~bootstrap/scss/bootstrap"

**Comment by JoshuaVSherman** [comment]
@Unesdala also please test the CollegeLutheran changes you made and fix those too so that they match this project (once you get it all figured out and working), thanks

**Pull request closed without merging**